src/stp/tactics/passive/BlockBall.cpp

```diff
@@ -53,6 +53,9 @@ Vector2 BlockBall::calculateTargetPosition(const world::view::BallView &ball, Ve
             // Default distance of 0.5m. If the ball is closer than that to the enemy, stand right in front of the ball instead.
             distance = std::min(0.5, targetToBall.length() - control_constants::ROBOT_RADIUS);
             break;
+        case BlockDistance::PARTWAY:
+            distance = targetToBall.length() * 0.4;
```

Comment:
When should PARTWAY be used instead of HALFWAY? Is there really a significant difference between 0.4 or 0.5 times the vector length?